pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[project]
+name = "splat"
+# Should be synced with src/splat/__init__.py
+version = "0.21.0"
+description = "A binary splitting tool to assist with decompilation and modding projects"
+authors = [
+    { name="Example Author", email="author@example.com" },
+]
+# Do we want a maintainers?
+# maintainers = = [
+# ]
```

Comment:
Just remembered this. Do we even want it? it isn't required to publish a package

Reply:
ah yeah I think it's fine to omit it